packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -350,9 +350,6 @@ module.exports = {
       },
     ],
 
-    // disallow space between function identifier and application
-    'no-spaced-func': 'error',
-
```

Comment:
actually on second thought, let's leave this here, set to "off", with a comment explaining that it's deprecated in favor of the newer rule.

Reply:
Yeah, that makes sense